components/listing-management.tsx

```
@@ -0,0 +1,460 @@
+                <h3 className="text-lg font-medium">No inactive listings found</h3>
+                <p className="text-muted-foreground mt-1">Try adjusting your search or filters</p>
+              </div>
+            )}
+          </div>
+        </TabsContent>
+      </Tabs>
+    </div>
+  )
+}
+
+function ListingCard({ listing }: { listing: any }) {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)